### PR TITLE
Udayan/dependency bears

### PR DESCRIPTION
**Pull request by Udayan12167**

### CODE REVIEW

**Comment by Asnelchristian** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/673/commits/2d81ca47047a5f36abb834070e6eaacfad114106

**Comment by Asnelchristian** [comment]
@rultor merge

**Comment by Udayan12167** [comment]
reack 4130233 387252f 2a3cd7f

**Comment by Udayan12167** [comment]
@rultor merge

File: bears/general/IndentationBear.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class IndentationBear(LocalBear):
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
+    BEAR_DEPS = {AnnotationBear}  # pragma: no cover
```
Comment (sils1297) [inline bears/general/IndentationBear.py]: why is the pragme needed? That should be removable

**Pull request merged**